<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort485</class>
 <widget class="QMainWindow" name="SerialPort485">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPort485</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{ 
border: 2px solid green; 
font: bold large &quot;New Century Schoolbook&quot;;
font-size: 30px;
text-align:center;
}

QPushButton
{
color:blue;
font-size: 20px;
font-family: &quot;New Century Schoolbook&quot;;
font-weight:bold;
text-align:center;
}
QTextBrowser
{
font-size: 20px;
font-family: &quot;New Century Schoolbook&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>196</width>
      <height>316</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>New Century Schoolbook</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-family: &quot;New Century Schoolbook&quot;;
font-size: 20px;
text-align:center;
}</string>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>端口号</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>155</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>155</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>205</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>205</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>255</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>255</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>275</y>
      <width>561</width>
      <height>356</height>
     </rect>
    </property>
    <property name="title">
     <string>接收数据</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>40</y>
       <width>536</width>
       <height>256</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>305</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_test">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>305</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>test</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>25</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>发送数据</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>190</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>40</y>
       <width>526</width>
       <height>136</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>185</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>640</y>
      <width>1166</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>20</y>
       <width>761</width>
       <height>51</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
font-size: 30px;
font-family: &quot;New Century Schoolbook&quot;;
font-weight:bold;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>126</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>30</y>
       <width>126</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>25</y>
      <width>391</width>
      <height>606</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-family: &quot;New Century Schoolbook&quot;;
font-size: 30px;
text-align:center;
}</string>
    </property>
    <property name="title">
     <string>参数显示</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>65</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>X轴</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>225</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Y轴</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>315</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>Z轴</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>50</y>
       <width>176</width>
       <height>61</height>
      </rect>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>135</y>
       <width>176</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>225</y>
       <width>176</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>176</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>435</y>
       <width>116</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>自动读取</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>435</y>
       <width>116</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>停止读取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>65</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>150</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>mm/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>240</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>mm/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>325</y>
       <width>76</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>mm/s</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>350</y>
      <width>196</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-family: &quot;New Century Schoolbook&quot;;
font-size: 25px;
font-weight:bold;
text-align:center;
}
</string>
    </property>
    <property name="title">
     <string>计数</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>45</y>
       <width>61</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="send_cout">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>45</y>
       <width>72</width>
       <height>46</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active/>
       <inactive/>
       <disabled/>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="receive_count">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>115</y>
       <width>72</width>
       <height>46</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active/>
       <inactive/>
       <disabled/>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>115</y>
       <width>61</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>接收</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>190</y>
       <width>136</width>
       <height>46</height>
      </rect>
     </property>
     <property name="text">
      <string>计数清零</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
